packages/mongo/mongo_driver.js

```diff
@@ -949,7 +949,23 @@ Cursor.prototype.observe = function (callbacks) {
 
 Cursor.prototype.observeChanges = function (callbacks) {
   var self = this;
+  var methods = [
+    'addedAt',
+    'added',
+    'changedAt',
+    'changed',
+    'removedAt',
+    'removed',
+    'movedTo'
+  ];
```

Comment:
In minimongo it is made even uglier: https://github.com/meteor/meteor/blob/d6c4281eebe8fffcdeb415ccc361784f854126f4/packages/minimongo/minimongo.js#L381

I think a list is OK. I would not go wrapping everything because there might be other things there.